engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -73,6 +79,10 @@ public String apply(@Nullable Level input)
     }
   };
 
+  static {
```

Comment:
I prefer to avoid static initialization. It will not work for unit tests where both app master and containers/workers run in the same JVM but different threads.